src/westpa/core/h5io.py

```diff
@@ -77,6 +77,18 @@ def calc_chunksize(shape, dtype, max_chunksize=262144):
     return chunk_shape
 
 
+def tostr(b):
+    '''Convert a nonstandard string object ``b`` to str with the handling of the
+    case where ``b`` is bytes.'''
+
+    if b is None:
+        return None
+    elif isinstance(b, bytes):
+        return b.decode()
```

Comment:
```suggestion
        return b.decode('utf-8')
```
I suggest explicitly adding in 'utf-8' as the encoding to eliminate confusion.